src/main/java/com/google/sps/data/TestAdaptor.java

```diff
@@ -0,0 +1,43 @@
+package com.google.sps.data;
+
+import java.io.BufferedReader;
+import java.io.DataOutputStream;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+
+import com.google.gson.Gson;
+import com.google.gson.reflect.TypeToken;
+/** File to test the GetGovAdator file.
+ */
+public class TestAdaptor { 
+  public static void main(final String[] args) {
+    try {
+      NSWGovAdaptor adaptor = new NSWGovAdaptor();
+      HashMap<Integer, GovSiteDetails> Map = new HashMap<>();
```

Comment:
style: don't capitalize the first char of variable name. use `camelCase` or underscore instead, don't call it map, give it more meaningful name.

Reply:
This is Java! What's an underscore?
Also: https://google.github.io/styleguide/javaguide.html#s5-naming
"Local variable names are written in lowerCamelCase."

Reply:
haha, yea, i meant to use like `variable_name`, or some style prefer to make name start with  underscore `_class_member_var_name` for class member, but `_`  itself is reserved.